core/src/main/java/org/mskcc/cbio/portal/dao/DaoCancerStudy.java

```diff
@@ -34,6 +34,7 @@
 
 import java.sql.*;
 import java.text.*;
+import java.time.LocalDate;
```

Comment:
Confirmed that we are already committed to 1.8 -- portal will no longer build with 1.7 compiler.